src/utils/tx-history-filter.ts

```diff
@@ -64,12 +65,14 @@ export const txFilter = {
   },
 
   parseFormData: ({ type, ...formData }: TxFilterFormState): TxFilter => {
-    const DATE_FORMAT = 'yyyy-MM-dd'
-
     const filter: TxFilter['filter'] = _omitNullish({
       ...formData,
-      execution_date__gte: formData.execution_date__gte ? format(formData.execution_date__gte, DATE_FORMAT) : undefined,
-      execution_date__lte: formData.execution_date__lte ? format(formData.execution_date__lte, DATE_FORMAT) : undefined,
+      execution_date__gte: formData.execution_date__gte
+        ? startOfDay(formData.execution_date__gte).toISOString()
```

Comment:
I think depending on the timezone, this can still result in the previous day being selected. E.g. if the user is in GMT+2, the start of the day is being selected at 0:00:00 hours and with the conversion through `toISOString` it would result in the previous day because 2 hours are subtracted.

Reply:
My debugging shows the same. I'll revisit this later today.

Reply:
After looking into this deeper, it seems as though the trasaction list is returned in UTC but we don't the date labels to match the timezone:

![image](https://user-images.githubusercontent.com/20442784/188618325-75911b6f-e4c2-4243-8a11-a61a7d8eb24f.png)

Reply:
Regarding my original comment, after thinking more about it I think it makes sense to leave it as it is. Even if the query changes to the previous day through the conversion via `toISOString`, it would include results that are on the current day in the local timezone.